code/src/cljs/sixsq/nuvla/ui/edge_detail/views.cljs

```diff
@@ -730,7 +731,40 @@
                                         :position "bottom"}
                                :scales {:yAxes [{:type       "logarithmic"
                                                  :scaleLabel {:labelString "megabytes"
-                                                              :display     true}}]}}}]]]])]))
+                                                              :display     true}}]}}}]]]])
+     (when container-stats
+       [ui/GridRow {:centered true
+                    :columns  1}
+        [ui/GridColumn
+         [ui/Table {:compact "very", :selectable true, :basic "very"}
+          [ui/TableHeader
+           [ui/TableRow
+            [ui/TableHeaderCell "ID"]
+            [ui/TableHeaderCell "Container Name"]
+            [ui/TableHeaderCell "CPU %"]
+            [ui/TableHeaderCell "Mem Usage/Limit"]
+            [ui/TableHeaderCell "Mem %"]
+            [ui/TableHeaderCell "Net I/O"]
+            [ui/TableHeaderCell "Block I/O"]
+            [ui/TableHeaderCell "Status"]
+            [ui/TableHeaderCell "Restart Count"]]]
+
+          [ui/TableBody
+           (for [{:keys [id name cpu-percent mem-usage-limit
+                         mem-percent net-in-out blk-in-out
+                         container-status restart-count] :as cstat} container-stats]
```

Comment:
When binding is not used but useful for documentation append an _.
If it's not obvious you append an underscore `_cstat`.
Here `cstat` is not used but we can see that it's container-stats. So no need to have the `:as cstat`.